README.md

```diff
@@ -1,3 +1,5 @@
+[![Build status](https://ci.appveyor.com/api/projects/status/hpvrm5230qlay15m?svg=true)](https://ci.appveyor.com/project/erdemkiraz/dasher)
```

Comment:
You need to resolve the conflict. The file has been modified since you pulled it. Check the latest version of the README.md file from the master branch.